src/renderers/dom/shared/inputValueTracking.js

```diff
@@ -133,9 +133,10 @@ var inputValueTracking = {
       return false;
     }
     var tracker = getTracker(subject);
+    const isNode = (subject: any).nodeType === ELEMENT_NODE;
 
     if (!tracker) {
-      if (typeof (subject: any).tag === 'number') {
+      if (isNode || typeof (subject: any).tag === 'number') {
```

Comment:
For reference, `updateValueIfChanged` is called with a Fiber instance because of `getTargetInstForClickEvent` [ChangeEventPlugin.js#L217-L221](https://github.com/facebook/react/blob/b1768b5a48d1f82e4ef4150e0036c5f846d3758a/src/renderers/dom/shared/eventPlugins/ChangeEventPlugin.js#L217-L221)